src/oadp_resources/volsync/replication_source.py

```diff
@@ -8,3 +9,37 @@
 
 class ReplicationSource(NamespacedResource):
     api_group = ApiGroups.VOLSYC_API_GROUOP.value
+
+    class Label(Enum):
+        VOLUME_SNAPSHOT_BACKUP = "datamover.oadp.openshift.io/vsb"
+
+    class ReplicationSourceCondition(Enum):
+        class Type(Enum):
+            RECONCILED = "Reconciled"
+            SYNCHRONIZING = "Synchronizing"
+
+    def reconciled(self):
```

Comment:
Please Add Docstring.

Reply:
Explain, i don't think it is clear enough.
Please let's start always adding Docstrings

Reply:
Thanks for telling me, you are totally right!
It will improve the code, done.